includes/settings/js/src/components/fields/NumberFields.jsx

```diff
@@ -1,49 +1,79 @@
 /**
  * Additional form fields for the Number field type.
  */
-import React from "react";
+import React, { useContext, useState } from "react";
```

Comment:
```suggestion
import React, { useState } from "react";
```

Appears that `useContext` isn't used here.